src/app/stream-deck/util/packager.ts

```diff
@@ -69,8 +69,8 @@ function getCurrentSeason(
   const season = profile?.profile?.data?.currentSeasonHash
     ? defs?.Season.get(profile.profile.data.currentSeasonHash)
     : undefined;
-  const seasonPass = season?.seasonPassHash
-    ? defs?.SeasonPass.get(season.seasonPassHash)
+  const seasonPass = season?.seasonPassList[0].seasonPassHash
+    ? defs?.SeasonPass.get(season.seasonPassList[0].seasonPassHash)
```

Comment:
These would probably be safer as:
```suggestion
  const seasonPass = season?.seasonPassList[0]?.seasonPassHash
    ? defs?.SeasonPass.get(season.seasonPassList[0].seasonPassHash)
```